app/services/webuild_events_service.rb

```diff
@@ -0,0 +1,11 @@
+class WebuildEventsService
+  def get_events(event_url='https://webuild.sg/api/v1/events')
+    begin
+      response = RestClient.get event_url
+      response_json = JSON.parse(response.body)
```

Comment:
I don't think this temporary variable is justified, or what do you say?

We could perhaps do this?

```
response = RestClient.get(event_url)
JSON.parse(response)["events"]
```